ui/src/literature/components/FullTextLinksAction.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Map, List } from 'immutable';
+import PropTypes from 'prop-types';
+import { Button, Menu } from 'antd';
+
+import ListItemAction from '../../common/components/ListItemAction';
+import IconText from '../../common/components/IconText';
+import EventTracker from '../../common/components/EventTracker';
+import ExternalLink from '../../common/components/ExternalLink';
+import DropdownMenu from '../../common/components/DropdownMenu';
+import { removeProtocolAndWwwFromUrl } from '../../common/utils';
+
+function fullTextLinkToHrefDisplayPair(fullTextLink) {
+  const href = fullTextLink.get('value');
+  const description = fullTextLink.get('description');
+  const display = description || removeProtocolAndWwwFromUrl(href);
+  return Map({ href, display });
+}
+
+class FullTextLinksAction extends Component {
+  static renderIcon() {
+    return <IconText text="pdf" type="download" />;
+  }
+
+  renderDropdown() {
+    const { fullTextLinks } = this.props;
+    const hrefDisplayPairs = fullTextLinks.map(fullTextLinkToHrefDisplayPair);
+    return (
+      <DropdownMenu title={<Button>{FullTextLinksAction.renderIcon()}</Button>}>
```

Comment:
can we just put the logic on ``renderIcon`` and rename it to something like ``renderIconButton``?

Reply:
nope, because it's not always button.. when it's single we need `ExternalLink` that wraps the icon.